webrtc-stats.html

```diff
@@ -1066,6 +1068,26 @@
                   section 6.4.1.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>packetsRetransmitted</code></dfn> of type <span class="idlMemberType"><a>unsigned
+                long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Total number of RTP packets retransmitted by the sender on receiving a retransmission request (e.g., NACK) from the receiver for this SSRC. The retransmitted packet may be sent as defined in
+                  [[!RFC4588]].
```

Comment:
@taylor-b any further comments? or should I just close the PR and the corresponding issue?